code-studio/package.json

```diff
@@ -18,27 +18,32 @@
     "lint-css": "sass-lint src/css/*css",
     "lint-js": "jshint src/js/*.js",
     "build": "npm run copy-assets && npm run build-js && npm run build-css",
+    "build:dist": "npm run lint && npm test && npm run clean && npm run copy-assets && npm run build-css && npm run build-js:min && npm run uglify-js",
```

Comment:
I'm interested in writing a script for this, but that will be a little clunky while I'm still having trouble being sure the JS build process is done running thanks to problems with `factor-bundle`.  It works here because the `&&` operator waits for the process to exit, which includes all of factor-bundle's work.  I'd like to leave this for now (I wouldn't expect it to grow any more!) and clean it up once we think using callbacks in Node is reliable.